gateway/proxy_eth.go

```diff
@@ -200,6 +200,14 @@ func (gw *Node) EthGetBlockByNumber(ctx context.Context, blkNum string, fullTxIn
 	return gw.target.EthGetBlockByNumber(ctx, blkNum, fullTxInfo)
 }
 
+func (gw *Node) EthGetTransactionByBlockHashAndIndex(ctx context.Context, blkHash ethtypes.EthHash, txIndex ethtypes.EthUint64) (*ethtypes.EthTx, error) {
+	return gw.target.EthGetTransactionByBlockHashAndIndex(ctx, blkHash, txIndex)
```

Comment:
needs rate limiting and tipset lookback checking

see `EthGetBlockTransactionCountByHash` in here for how to do this

Reply:
@virajbhartiya You have done something similar in past PRs as well. Would reccomend to look back at the work you've done for a sanity check to see what's missing in the PRs you raise.